lib/reek/cli/options.rb

```diff
@@ -10,28 +8,40 @@ module CLI
     # Parses the command line
     #
     # See {file:docs/Command-Line-Options.md} for details.
+    #
+    # :reek:TooManyInstanceVariables: { max_instance_variables: 6 }
```

Comment:
Those 2 smells were kind of there before but hidden since we were using the OpenStruct as "options" attribute.